src/Persistence/Array_.php

```diff
@@ -132,17 +128,15 @@ public function tryLoad(Model $m, $id, $table = null)
      *
      * @param Model $m
      * @param mixed $table
-     *
-     * @return array|false
      */
-    public function tryLoadAny(Model $m, $table = null)
+    public function tryLoadAny(Model $m, $table = null): ?array
```

Comment:
Looks like $table can be only string, so we can set type here. I doubt that somebody will set table name as integer.